float32 stick_left_x
float32 stick_left_y
float32 stick_right_x
float32 stick_right_y
float32 trigger_left
float32 trigger_right
bool button_a
bool button_b
bool button_x
bool button_y
bool bumper_left
bool bumper_right
bool button_back
bool button_xbox
bool button_start
bool button_left_stick
bool button_right_stick
bool d_pad_up
bool d_pad_down
bool d_pad_left
bool d_pad_right